test/operations/common/BitBucketServerRepoRef.test.ts

```diff
@@ -93,16 +88,73 @@ describe("BitBucketServer support", () => {
             "refs/heads/master");
     });
 
-    it("should create pr without default reviewers", async () => {
+    it("should create pr with default reviewers and merge manually supplied ones", async () => {
         const mock = new MockAdapter(axios);
 
-        mock.onGet("https://bitbucket.organisation.co.za/rest/api/1.0/projects/a-project/repos/test-app")
-            .reply(200, { id: 1 });
+        getRepo(mock);
+        getRepoDefaultReviewers(mock, ["johndoe"]);
+        mock.onPost("https://bitbucket.organisation.co.za/rest/api/1.0/projects/a-project/repos/test-app/pull-requests")
+            .reply(config => {
+                const postData = JSON.parse(config.data);
+                assert((postData.reviewers as any[]).length === 2);
+                assert((postData.reviewers as any[])[0].user.name === "johndoe");
+                assert((postData.reviewers as any[])[1].user.name === "janedoe");
+                return [201, {}];
+            });
 
-        // tslint:disable-next-line:max-line-length
-        mock.onGet("https://bitbucket.organisation.co.za/rest/default-reviewers/1.0/projects/a-project/repos/test-app/reviewers?sourceRepoId=1&targetRepoId=1&sourceRefId=refs/heads/thing1&targetRefId=refs/heads/master")
-            .reply(200, []);
+        const bitbucketServerRepoRef = new BitBucketServerRepoRef("https://bitbucket.organisation.co.za", "a-project", "test-app");
+        return bitbucketServerRepoRef.raisePullRequest(
+            BitBucketServerCredentials, "Add a thing",
+            "Mr Peanut Butter goes woof", "refs/heads/thing1",
```

Comment:
That does not even make sense.

Reply:
Is Mr. Peanut Butter even a dog?!?

Reply:
What am I supposed to believe here?

Reply:
Mr. Peanut Butter, the dog (presumably), first made his appearence in some BitBucket tests a while back.  I chose to continue the tradition.